text/0000-signed-transaction-payload/0000-signed-transaction-payload.md

```diff
@@ -0,0 +1,825 @@
+                            </details>
+                            <details>
+                                <summary>Signed Data Payload</summary>
+                                <blockquote>
+                                Describes a payload containing signed data (and its corresponding Ed25519 public key and signature) without
+                                any specification what the data means.
+                                </blockquote>
+                                <table>
+                                    <tr>
+                                        <th>Name</th>
+                                        <th>Type</th>
+                                        <th>Description</th>
+                                    </tr>
+                                    <tr>
+                                        <td>Payload Type</td>
+                                        <td>varint</td>
+                                        <td>
+                                        Set to <strong>value 3</strong> to denote an <i>Signed Data</i> payload.
+                                        </td>
+                                    </tr>
+                                    <tr>
+                                        <td>Data</td>
+                                        <td>ByteArray</td>
+                                        <td>The data of the <i>Signed Data</i> payload.</td>
+                                    </tr>
+                                    <tr>
+                                        <td>Public Key</td>
+                                        <td>ByteArray[32]</td>
+                                        <td>The Ed25519 public key used to verify the signature.</td>
+                                    </tr>
+                                    <tr>
+                                        <td>Signature</td>
+                                        <td>ByteArray[64]</td>
+                                        <td>The signature signing the data.</td>
+                                    </tr>
+                                </table>
+                            </details>
+                            <details>
+                                <summary>Indexation Payload</summary>
+                                <blockquote>
+                                Describes a payload which is used to instruct the node to index the <i>Message</i>
+                                of which it is part of.    
+                                </blockquote>
+                                <table>
+                                    <tr>
+                                        <th>Name</th>
+                                        <th>Type</th>
+                                        <th>Description</th>
+                                    </tr>
+                                    <tr>
+                                        <td>Payload Type</td>
+                                        <td>varint</td>
```

Comment:
why use varint if you set the rule to be < 127? why not uint8?

Reply:
This should clearly be consistent with the _Payload_ as introduced in #17. Since this representsthe layered payload approach.

There is no rule limiting this to 127 much larger values should be allowed.